### ISSUE TITLE
Grand Theft Auto V [BLUS31156] OpenGL Undefined Format Class for Depth Textures

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#15711]
### Quick summary

E RSX: Undefined format class for depth texture is not allowed

### Details

While using OpenGL to play GTAV, I tested as far back as 2020, I would receive when my character is above things (Like switching characters, standing on a cliff, flying a helicopter, climbing a building) a log stack spam of Undefined format class for depth texture is not allowed.
Performance is always around 3-11fps
Fog in distances will pop in and out of sight as if confused if it should exist or not

### Attach a log file

[RPCS3.log not using RSX buffers](https://github.com/user-attachments/files/15874730/RPCS3.log)
[RPCS3.log using write buffers](https://github.com/user-attachments/files/15875027/RPCS3.log)
[RPCS3.log using read and write buffers](https://github.com/user-attachments/files/15875102/RPCS3.log)

### System configuration

AMD Ryzen 7 7700X with integrated AMD Radeon Graphics running on driver 2.0.302
NVIDIA GeForce GTX 1660 SUPER running on driver 555.99.0.0

**Comment by ghost** [comment]
Does this still happen?

**Comment by Darkhost1999** [comment]
Yes. Still has some graphical issues accompanying the depth textures not allowed message
